src/rules/order-properties.ts

```diff
@@ -58,10 +58,10 @@ export const rule = createRule({
 				const json = JSON.parse(text) as Record<string, unknown>;
 
 				const allKeys = Object.keys(json);
-				const nonStandardKeys = allKeys.filter(
-					(key) => !requiredOrder.includes(key),
-				);
-				const orderedNonStandardKeys = nonStandardKeys.sort();
+				const orderedNonStandardKeys = allKeys
+					.filter((key) => !requiredOrder.includes(key))
+					// eslint-disable-next-line unicorn/no-array-sort
+					.sort();
```

Comment:
-1 from me on enabling this rule pending https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2735#issuecomment-3637969180. Out of the common benefits of the rule:

* Performance: we don't deal with large enough sets of data for that to be a bottleneck
* Testing: we already have relatively high test coverage, so it's not likely to catch many bugs

(obviously linting is nice as an earlier line of defense - I'm saying the benefits are severely reduced, not eliminated)

Reply:
Gotta love the insta-close after your comment 😭 .  In this case, we should probably turn the rule off, as they suggested?